apps/juicebox_stream/test/juicebox_stream/youtube/client_test.exs

```diff
@@ -0,0 +1,17 @@
+defmodule JuiceboxStream.Youtube.ClientTests do
+  use ExUnit.Case, async: true
+  alias JuiceboxStream.Youtube.{Client, Video}
```

Comment:
Most of the time you are using the multiple single line alias/require/import/use directives but here you are using the multi-alias/require/import/use syntax